samples/TagHelperSample.Web/Controllers/HomeController.cs

```diff
@@ -33,7 +33,7 @@ public IActionResult Index()
         // GET: /Home/Create
         public IActionResult Create()
         {
-            ViewBag.Items = _items;
+            ViewData["Items"] = _items;
```

Comment:
Because we've moved completely away from demonstrating the horrendously poor performance of `ViewBag`. Look at any current ASP.NET Core project template.